cmd/dashboard/command_history.js

```diff
@@ -24,6 +29,16 @@ class CommandHistory {
     this.pointer++;
     return this.history[this.pointer];
   }
-}
 
+  loadHistory() {
+    if (fs.existsSync(this.cmdHistoryFile)) {
+      fs.readFileSync(this.cmdHistoryFile)
+        .toString()
+        .split('\n')
+        .reverse()
+        .forEach((cmd) => { this.addCommand(cmd); })
+    }
+  }
```

Comment:
You can remove this as the console module is the one to handle the file.

Reply:
Hi @jrainville. That's a tricky one. I was debugging this for a while since the internal dashboard history traversal triggered by up/down keys was not showing up the items and the reason is that the dashboard initialization happens before console service is started. 

To be more specific,

https://github.com/embark-framework/embark/blob/develop/cmd/cmd_controller.js#L109

and console service is started a bit below

https://github.com/embark-framework/embark/blob/develop/cmd/cmd_controller.js#L137

instead of implementing lazy initialization of the history, I decided to leave it in the current state as it  does not seem to do any harm to just pre-initialize dashboard internal history cache and the history is available immediately. No reading files is needed at any point later. Repl console and 'history' command in the console and dashboard are fully event based. I hope that is fine for you. If not we have to figure out another solution.

Btw. Travis & Appveyor are green again! :)